test/validation/api/classification/odp_classification_common.c

```diff
@@ -496,6 +508,15 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 			ODPH_ERR("odph_udp_tcp_chksum failed\n");
 			return ODP_PACKET_INVALID;
 		}
+	} else if (pkt_info.l4_type == CLS_PKT_L4_AH) {
+		ah->next_header = ODPH_IPV4;
+		ah->ah_len = ODPH_AHHDR_LEN;
+		ah->pad = 0;
+		ah->spi = 0;
+		ah->seq_no = 0;
```

Comment:
It would be better to set this to 1 (and likewise for ESP) since zero sequence number should never appear in the wire, which some ODP implementation might assume.